disnake/ui/action_row.py

```diff
@@ -236,10 +294,47 @@ def components_to_dict(components: Components) -> List[ActionRowPayload]:
 
             else:
                 raise ValueError(
-                    "components must be an Item, a list of ActionRows or a list of Items"
+                    "components must be a WrappedComponent, a list of ActionRows "
+                    "or a list of WrappedComponents"
                 )
 
     if auto_row.width > 0:
         action_rows.append(auto_row.to_component_dict())
 
     return action_rows
+
+
+def components_to_rows(components: Components) -> List[ActionRow]:
+    if not isinstance(components, list):
+        components = [components]
+
+    action_rows: List[ActionRow] = []
+    auto_row = ActionRow()
+
+    for component in components:
+        if isinstance(component, WrappedComponent):
+            try:
+                auto_row.append_item(component)
+            except ValueError:
+                action_rows.append(auto_row)
+                auto_row = ActionRow(component)
+        else:
+            if auto_row.width > 0:
+                action_rows.append(auto_row)
+                auto_row = ActionRow()
+
+            if isinstance(component, ActionRow):
+                action_rows.append(component)
+
+            elif isinstance(component, list):
+                action_rows.append(ActionRow(*component))
+
+            else:
+                raise ValueError(
+                    "components must be an Item, a list of ActionRows or a list of Items"
```

Comment:
```suggestion
                    "components must be an Item, a list of ActionRow or a list of Items"
```
Shouldn't we use the new `WrappedComponent`? (plus this suggestion, which is basically to avoid confusions like: "oh ok, I need to use ActionRows(,..)", kind of weird but you know)